client/client_test.go

```diff
@@ -123,6 +133,23 @@ func createRepoAndKey(t *testing.T, rootType, tempBaseDir, gun, url string) (
 	return repo, rootPubKey.ID()
 }
 
+// creates a new notary repository with the same gun and url as the previous
+// repo, so that it can be used to pull the trust data the original repo pushed
+func newRepoToTestRepo(t *testing.T, existingRepo *NotaryRepository) *NotaryRepository {
```

Comment:
Is this to avoid deleting files in a tmpDir mid way to replicate a clean Repository state?